frontend/components/users/profile/ProfilePhotoSelectionModal.tsx

```
@@ -0,0 +1,178 @@
+  if (!isOpen) return null;
+
+  return (
+    <Dialog open={isOpen} onClose={handleClose} className="relative z-50">
+      <DialogBackdrop className="fixed inset-0 bg-black/50" />
+
+      <div className="fixed inset-0 z-10 overflow-y-auto">
+        <div className="flex min-h-full items-center justify-center p-4">
+          <DialogPanel className="relative w-full max-w-3xl max-h-[90vh] flex flex-col bg-white rounded-lg shadow-xl">
+            <div className="flex items-center justify-between p-6 border-b border-core-outline">
+              <h2 className="text-2xl font-semibold text-core-text">Change Profile Picture</h2>
+              <button onClick={handleClose} className="text-gray-400 hover:text-gray-600" disabled={isUploading}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)